config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -6,6 +6,15 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: machines.cluster.k8s.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.providerID
+    description: Provider ID
+    name: ProviderID
+    type: string
+  - JSONPath: .status.phase
+    description: status of machine
```

Comment:
```suggestion
    description: Phase represents the current machine status (e.g. Pending, Running, Terminating, Failed, etc.).
```

Reply:
ok, will do

Reply:
It looks like this comment still applies? Maybe a result of the update to remove the RBAC change?